t80s_reduce/t80s_gen_download/main.py

```diff
@@ -129,3 +129,7 @@ def main(argv):
         fp.write(download_string)
 
     return 0
+
```

Comment:
In this package I have not used this style for calling the scripts. Instead I create a "__main__.py" which is responsible for calling the main function. This way you can call the package name directly from python to run the script. 

So, remove the following lines.